AC-784 update about offline operations (#45)
openmrs/openmrs-android-client-user-guide: provider-management.md

On opening the Provider Management, you are greeted with a screen with active providers and a **Floating Action Button**. **Click** on the **Floating Action Button** and it will take you to new screen asking for Provider details. 

![Confirmation while creating new Provider](assets/provider_create_confirmation.png)

> The details are then matched with other existing Providers and a dialog box is shown for confirmation.If the device is online the provider gets synced to the servers otherwise your created provider gets saved offline and gets synced later.

Reviewer: ```suggestion
The details are then matched with other existing Providers and a dialog box is shown for confirmation. If the device is online, the provider is synced to the server, otherwise the created provider is saved offline and will be synced when your device is back online.
```

#### Update Providers

> To update a Provider, **swipe left** on the Provider to reveal additional actions (Update and Delete). **Click** on the **edit** button. It will take you to a screen where the details can be edited.If the device is online the provider gets synced to the servers otherwise your edited provider gets saved offline and gets synced later.

Reviewer: ```suggestion
To update a Provider, **swipe left** on the Provider to reveal additional actions (Update and Delete). **Click** on the **edit** button. It will take you to a screen where the details can be edited. If the device is online the provider gets synced to the servers otherwise your edited provider gets saved offline and gets synced later.
```
